### ISSUE TITLE
Revise default shortcuts

### ISSUE THREAD

**Issue by matthiasgeihs** [continuedev/continue#2133]
### Validations

- [ ] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [ ] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

Cmd+L is the default shortcut for line selection, which I use frequently.
Having Continue overwrite this shortcut doesn't work for me.

Moreover, the shortcut seems to be hardcoded in a couple of places. When I set a different shortcut, the tooltip hints will still mention Cmd+L as the shortcut (even though this is not longer true).

### Solution

I'd prefer a different default shortcut for adding current selection to the chat context, e.g., Cmd+Shift+I.

**Comment by sestinj** [comment]
Hi @matthiasgeihs! Thank you for this feedback. I would recommend re-mapping the shortcut to something that works better for you. While I'm sure this was frustrating, we've learned (through tons of trial and error before eventually landing here) that there is no shortcut that is both ergonomic _and_ won't bother at least some subset of users. We've taken what we believed to be the best route and don't plan to change right now.

You're right about the hardcoding and this definitely isn't ideal. It doesn't look like there's any reliable way of getting the list of keyboard shortcuts for commands from the VS Code extension API, so we may be stuck showing the default value

**Comment by matthiasgeihs** [comment]
fair enough, i'll leave it open for a bit, to see if others encounter a similar problem, or want to add to the discussion here. feel free to close this any time you feel appropriate.

**Comment by appleseed-iii** [comment]
+1

**Comment by shiraleeana** [comment]
+1 I find it super frustrating to have to change the default workflow when a new extension is added and remaps a default shortcut. My beef is with CMD + SHIFT + L to add a cursor to all instances of a selection.

How would one remap the shortcut in Continue to something else? Have tried looking for documentation but couldn't seem to find it, sorry!

I've had to disable Continue for now 😿

**Comment by jester6san** [comment]
+1 for same reasons as described by previous users and changing the default vscode shortcuts is not an option for me.

**Comment by willvarney** [comment]
+1 for the same reason, it took me way too long (facepalming myself) to figure out why my CMD + L wasn't working :/

**Comment by matthiasgeihs** [comment]
i think this is important. The Cursor IDE actually has the same issue. Why not simply pick a default extension shortcut that doesn't overwrite a default VS Code shortcut?

**Comment by Anmol-Sharma** [comment]
This issue was really irritating, so I've found a solution to this problem. We can directly remap the extension shortcuts in vscode without modifying the original shortcuts. For example I've mapped `cmd/ctrl + L` and `cmd/ctrl + shift + L` separately 👇 to my own liking using vscode keyboard shortcuts settings (Cmd + K, Cmd + S on MacOS).


![Image](https://github.com/user-attachments/assets/84ddee6f-5820-4311-82ad-aab7f28a1436)

Hope someone looking for a solution finds this useful.

**Comment by Ximaz** [comment]
The thing is when remapping some keys, the tooltip is lying with the old shortcuts. For instance, I remapped `shift+cmd+l` to `cmd+@`, but the tooltip keeps showing `cmd+L` (`shift+cmd+l`), even though I restarted VSCode.

<img width="871" height="103" alt="Image" src="https://github.com/user-attachments/assets/c670f852-9c2d-48a5-aec1-423dbae204d1" />

<img width="871" height="135" alt="Image" src="https://github.com/user-attachments/assets/558035c6-6cd5-46a2-8fe2-9fe6823be440" />

The remapping works though, I can use `cmd+@` instead of `shift+cmd+l` as expected. The UI is just not helping here.
